Sedah/Assets/Scripts/Character/PlayerMovement.cs

```diff
@@ -14,18 +14,23 @@ public class PlayerMovement : NetworkBehaviour
         public float movementSpeed = 30;
         private void Start()
         {
-            //// This script should only exist on the local player
-            //if (NetworkClient.localPlayer.netId != base.netId)
-            //{
-            //    Destroy(this);
-            //}
+            if (isLocalPlayer)
+            {
+                agent = GetComponent<NavMeshAgent>();
```

Comment:
We should only set the agent once. Also it should probably only be set on the server, since the client can't control it directly

Reply:
sounds goood